### PR TITLE
Add Compiler/Language Changes in 2.068.

### PR DESCRIPTION
**Pull request by 9rnsr**

### CODE REVIEW

**Comment by 9rnsr** [comment]
@MartinNowak Ping.

File: changelog.dd [hunk 1]
```diff
@@ -4,7 +4,120 @@ $(D_S D Change Log,
...
+
+        void test() { void* x; auto y = cast(void delegate())x; }
+        // Since 2.067: it had reported internal compiler error "Error: e2ir: cannot cast ..."
+        //  From 2.068: it reports proper front-end error.
+        ---
+)
+
+)
+
+$(BR)$(BIG $(LNAME2 list2068, List of all bug fixes and enhancementsin D 2.068:))
```
Comment (aG0aep6G) [inline changelog.dd]: Typo: "enhancementsin" -> "enhancements in"

Comment (9rnsr) [inline changelog.dd]: Oops, thanks!

**Comment by MartinNowak** [comment]
How did I miss that one?

**Pull request merged**